### PR TITLE
More environment variables check

### PR DESCRIPTION
**Pull request by LovelyMint**
- More environment variables check

### CODE REVIEW

File: src/providers/Environment.ts [hunk 1]
```diff
@@ -1,24 +1,54 @@
 import * as path from 'path';
 import * as dotenv from 'dotenv';
+import validator from 'validator';
 
 import Logger from '../libs/Logger';
 
-const requiredENV = ['NODE_ENV', 'DATABASE_URL', 'DISCORD_TOKEN', 'PRIVATE_BOT', 'OSU_API_KEY'];
```
Comment (YuzuZensai) [inline src/providers/Environment.ts:6]: I think the osu! module is going to crash if the API key isn't defined. Make change to the osu! module to handle this too

**Review by YuzuZensai — CHANGES_REQUESTED** [review]

File: src/providers/Environment.ts (hunk 1 above)
Comment (YuzuZensai) [reply src/providers/Environment.ts:6]: DEVELOPER_IDS too, but I already pushed a change that will handle this

Comment (LovelyMint) [reply src/providers/Environment.ts:6]: Done~!

File: src/discord/osu.ts [hunk 2]
```diff
@@ -111,7 +123,7 @@ export default class osu extends DiscordModule {
                         embeds: [EMBEDS.INVALID_USER_MENTIONED(data)]
                     });
 
-                let result = await osuAPI.client.getUser({ u: user });
+                let result = await osuAPI.client!.getUser({ u: user });
```
Comment (YuzuZensai) [inline src/discord/osu.ts:126]: Not the best solution here, but this module will be rewrite in the future

**Review by YuzuZensai — APPROVED** [review]

**Pull request merged**